cmd/openshift-install/main.go

```diff
@@ -40,6 +40,19 @@ func main() {
 	installerMain()
 }
 
+// Drop all terraform environment variables that the user may have passed
+// so they can't use terraform directly.
+func cleanEnvironment() {
+	environ := os.Environ()
+	for _, env := range environ {
+		splits := strings.Split(env, "=")
+		key := splits[0]
+		if strings.HasPrefix(key, "TF_") {
```

Comment:
Doesn't this also unset `TF_VAR_libvirt_master_memory` and `TF_VAR_libvirt_master_vcpu`? We need a way to do that.

Reply:
If the memory requirements are inadequate you need to fix the template. Terraform is not for users.

Reply:
Yeah, we can have the right amount for memory in the template i guess. What about the VCPU? I guess it's time we implement suggestion of @cgwalters to autodetect that.